configs/minimal.yaml

```diff
@@ -158,6 +158,9 @@ NUMBER_OF_COLUMNS: 128
 MAX_CELLS_IN_EXTENDED_MATRIX: 768
 DATA_COLUMN_SIDECAR_SUBNET_COUNT: 32
 MAX_REQUEST_DATA_COLUMN_SIDECARS: 16384
+SAMPLES_PER_SLOT: 8
+CUSTODY_REQUIREMENT: 1
+TARGET_NUMBER_OF_PEERS: 70
```

Comment:
Do we need the target number of peers in the config? 

Currently I think most clients expose this via a CLI flag. Each client sets their own defaults based on node-resources and misc tradeoffs with performance. I think we probably still want to leave this up to client implementations to set